src/tests/tracing/eventpipe/common/IpcSocket.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Net;
+using System.Net.Sockets;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Tracing.Tests.Common
+{
+    internal class IpcSocket : Socket
```

Comment:
if `IpcUnixDomainSocket` isn't needed and Socket be used directly, we can remove this.